valorant-tracker/src/App.js

```diff
@@ -1,24 +1,15 @@
 import logo from './logo.svg';
 import './App.css';
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import Home from './pages/home/Home';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Valorant Tracker
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+  <BrowserRouter>
```

Comment:
Form will be on the home page but there are no plans currently to navigate away from it. All information from API will be displayed on the same page of the form. We may keep React Routing for now. Continue to be mindful of component names under this as all other works in the nearby future will be directly in the Home route.